lib/assessment.md

```diff
@@ -22,14 +22,29 @@ Using this format will make it easier for us to grade you, so that we can get ev
 
 When we say that the web is a service, what does that mean? Explain the interaction in terms of clients and servers.
 <!-- your answer starts here -->
-
+The sevice
+Client sends a request to the server. The server takes the request and sends a response.
```

Comment:
This is correct. Just keep in mind that this is what a service _is_ : two parties acting together, sending and responding to requests according to certain rules.